pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -345,7 +345,7 @@ func buildLocalityLbEndpoints(env *model.Environment, proxyNetworkView map[strin
 		lbEndpoints[locality] = append(lbEndpoints[locality], ep)
 	}
 
-	LocalityLbEndpoints := make([]endpoint.LocalityLbEndpoints, len(lbEndpoints))
+	var LocalityLbEndpoints []endpoint.LocalityLbEndpoints
```

Comment:
good catch, i prefer 

localityLbEndpoints := make([]endpoint.LocalityLbEndpoints, 0, len(lbEndpoints))

Reply:
note: lower case`localityLbEndpoints `

Reply:
Oh yeah, that is better. Thanks, fixed both of those now